velox/connectors/hive/storage_adapters/hdfs/HdfsReadFile.h

```diff
@@ -61,6 +103,7 @@ class HdfsReadFile final : public ReadFile {
   hdfsFS hdfsClient_;
   hdfsFileInfo* fileInfo_;
   std::string filePath_;
+  folly::ThreadLocal<HdfsFile> file_;
```

Comment:
@JkSelf Let's use the PIMPL idiom here. It is not ideal to expose implementation headers here.
See for example: https://github.com/majetideepak/velox/blob/main/velox/connectors/hive/storage_adapters/abfs/AbfsReadFile.h

Reply:
@majetideepak Yes. Updated. Can you help to review again? Thanks.